CHANGES.rst

```diff
@@ -17,6 +17,9 @@ debops.apache v0.1.0 - unreleased
 Added
 ~~~~~
 
+- Added a apache__redirect_to_https to control the role's default behaviour for
+  redirecting to https. [muelli_]
```

Comment:
You are not yet defined in [80post.rst](https://github.com/debops/docs/blob/master/docs/includes/80post.rst), feel free to add yourself to the file and run `make includes/global.rst` to get docs working.

`Added a apache__redirect_to_https to control` →

```
Add :envvar:`apache__redirect_to_https to control`
```

or

```
Add ``apache__redirect_to_https to control``
```

and `debops-optimize` will take care of it.